cmd/state-svc/main.go

```diff
@@ -185,6 +185,7 @@ func runForeground(cfg *config.Instance, an *anaSync.Client, auth *authenticatio
 
 	logFile := logging.FilePath()
 	logging.Debug("Logging to %q", logFile)
+	logging.StartRotateLogTimer()
```

Comment:
This should be closed at shutdown, otherwise, we're shutting down with an active goroutine running.